llvm/lib/IR/RepoTicket.cpp

```diff
@@ -139,12 +139,18 @@ static bool updateDigestUseDependenciesAndContributions(
   GOHash.update('T');
   auto State = AccumulateGODigest(GO, GOHash, GOIMap);
   Changed = Changed || State.Changed;
```

Comment:
Change IS false here so this is effectively Changed = State.Changed.

So this can be combined with with the above to be:
`bool Changed = State.Changed;`

Reply:
Fixed, thanks.